.travis.yml

```diff
@@ -5,12 +5,17 @@ go:
 - 1.8
 - 1.9
 - tip
+before_install:
+  - go get -t -v ./...
 before_script:
 - go get -u github.com/alecthomas/gometalinter
 - gometalinter --install
 script:
 - gometalinter --disable=golint ./...
 - go test -cover -bench=. -v -race ./...
+- ./go.test.sh
```

Comment:
Where does this file come from? Is it generated in a previous line? I don't see it in the repo...

Reply:
@mbernier: Forgot to add it, updated